__tests__/integration/utils/test-utils-integration.js

```diff
@@ -72,11 +74,14 @@ export class TestUtils {
   }
 
   /**
-   * Returns the Supertest `request` object for this application
-   * @returns {Test}
+   * Whenever the tests need to check for a wallet/address balance, there should be a small pause
+   * to allow for the Fullnode's Websocket connection to update the Wallet Headless' local caches.
+   *
+   * The delay period here should be optimized for this purpose.
+   * @returns {Promise<void>}
    */
-  static get request() {
-    return request;
+  static async pauseForWsUpdate() {
+    await TestUtils.delay(1000);
```

Comment:
Maybe this should be a env variable

Reply:
Moved it to the config file, together with the transaction log settings.